docusaurus.config.js

```diff
@@ -30,7 +30,8 @@ const config = {
   trailingSlash: true,
 
   scripts: [
-    { src: "https://plausible.io/js/script.js", defer: true, "data-domain": "docs.metamask.io" },
+    { src: "https://plausible.io/js/script.js", defer: true, "data-domain": "docs.metamask.io" }, // legacy analytics
+    { src: "https://cmp.osano.com/AzZMxHTbQDOQD8c1J/a2e89f0e-f467-4542-bfea-30ea2c1a6648/osano.js", defer: true }, // cookie consent
```

Comment:
1. Please remove  " , defer: true "   
2. Link: https://docs.osano.com/installing-osano
3. **_"Important: The Osano.js should be placed without any async or defer attributes. Compliance cannot be guaranteed when using the above attributes. "_**
4. **_"Important: Osano will only be able to discover and block scripts that are loaded after it, therefore it must execute in the <head> before any other 3rd party or tracking content. "_**

Reply:
Thanks for the guidelines. I'll go ahead and remove the defer param.